LocalPlanner:
  max_vel_x: 0.3
  min_vel_x: 0.03
  max_vel_theta: 0.3
  min_in_place_vel_theta: 0.15
  meter_scoring: true
  acc_lim_theta: 0.15
  acc_lim_x: 0.15
  acc_lim_y: 0
  sim_time: 4.0
  vx_samples: 20
  vtheta_samples: 20
  pdist_scale_bias: 5
  gdist_scale_bias: 0.8
  dwa: false
  holonomic_robot: false
  yaw_goal_tolerance: 0.1
  xy_goal_tolerance: 0.15 